coralproject/talk
# Notifications: show "load new" button at top of notifications feed when available

## What does this PR do?

Shows a `Load new` notifications button at the top of the notifications feeds when new notifications come in while the tab/floating-tray/mobile-tray is already open.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

N/A

## How do I test this PR?

- post a comment as a user
- open the notifications tab/trays as a commenter
- with a different user, reply to the original commenter
- see that `Load New` appears at top of the notifications tab/tray for the first user
- click it
- see that the feed loads in the new notifications

## Were any tests migrated to React Testing Library?

No

## How do we deploy this PR?

Merge into epic branch.
